src/SFA.DAS.QnA.Api/Infrastructure/ApiError.cs

```diff
@@ -10,7 +10,7 @@ public class ApiError
 
         public string StatusDescription { get; private set; }
     
-        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
+        [JsonIgnore]
         public string Message { get; private set; }
```

Comment:
done

Reply:
Whereas Newtonsoft didn't care, [Text.Json won't serialise properties with private setters](https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/immutability?pivots=dotnet-6-0#non-public-property-accessors), so that needs to be sorted here